logger/modules/log_file_merger.py

```diff
@@ -0,0 +1,77 @@
+"""
+Merges log files.
+"""
+
+import os
+import pathlib
+from datetime import datetime
+
+from ..read_yaml.modules import read_yaml
+
+CONFIG_FILE_PATH = pathlib.Path(os.path.dirname(__file__), "config_logger.yaml")
+MERGED_LOGS_FILENAME = "merged_logs"
+
+
+def merge_log_files(log_file_directory: str) -> None:
+    """
+    Reads, sorts, and writes log files in the specified directory.
+
+    log_file_directory: The directory containing the log files to be merged.
+    """
+    # Read log files
+    log_files = [
+        file
+        for file in os.listdir(log_file_directory)
+        if file.endswith(".log") and file != f"{MERGED_LOGS_FILENAME}.log"
+    ]
+    log_entries = []
+    for log_file in log_files:
+        with open(os.path.join(log_file_directory, log_file), "r", encoding="utf-8") as file:
+            log_entries.extend(file.readlines())
+
+    # Sort log entries
+    log_entries.sort(key=lambda entry: datetime.strptime(entry.split(": ")[0], "%H:%M:%S"))
```

Comment:
Explain what this is doing. Might be worth pulling out into its own function for readability.